xcube_geodb/core/geodb.py

```diff
@@ -2391,7 +2391,12 @@ def unpublish_collection_from_group(
 
     def publish_database_to_group(self, group: str, database: Optional[str] = None) -> Message:
         """
-        Publishes the database to the given group.
+        Publishes the database to the given group and enables group users to write into the same database.
+        For example, if some user A creates the database `someproject`, users that share a group G with A are
+        allowed to create new collections in that database (like `someproject_insitudata`) if A publishes the database
+        to the group G. Group users shall not, however, be allowed to read or write any existing collection in the
```

Comment:
Hmm, why are the other users not allowed to read an existing collection in the shared database?

Reply:
> Hmm, why are the other users not allowed to read an existing collection in the shared database?

Methods should do one thing only ("do one thing well", https://en.wikipedia.org/wiki/Unix_philosophy). There is another dedicated method for allowing users to read existing collections in the shared database, i.e. `publish_collection_to_group`. `publish_database_to_group` has been missing before.